### PR TITLE
Added two more template switches to the examples

### PR DESCRIPTION
**Pull request by david-kalbermatten**
Just two more examples with icon-template and friendly name like 

```yaml
bedroom_privacy_mode_switch:
  friendly_name: Privacy Mode
  value_template: [...]
  turn_on: [...]
  turn_off: [...]
  icon_template: >-
    {% if state_attr('camera.bedroom_hd', 'privacy_mode') == 'on' %}
      mdi:eye-off-outline
    {% else %}
      mdi:eye-outline
    {% endif %}
```

### CODE REVIEW

File: examples/EXAMPLES_ENTITIES.md [hunk 1]
```diff
@@ -7,15 +7,19 @@ Below you can find some examples of usage of this custom component in template e
...
 
 ```yaml
+# configuration.yaml
+
 switch:
   - platform: template
     switches:
-      bedroom_privacy_mode:
+      # This is a switch to turn the privacy mode on/off
+      bedroom_privacy_mode_switch:
```
Comment (JurajNyiri) [inline examples/EXAMPLES_ENTITIES.md:21]: Please remove _switch from the name, its already under switch domain

File: examples/EXAMPLES_ENTITIES.md [hunk 2]
```diff
@@ -27,6 +31,47 @@ switch:
...
             entity_id: "camera.bedroom_hd"
             privacy_mode: "off"
+        icon_template: >-
+          {% if state_attr('camera.bedroom_hd', 'privacy_mode') == 'on' %}
+            mdi:eye-off-outline
+          {% else %}
+            mdi:eye-outline
+          {% endif %}
+      # This is a switch to turn the auto track mode on/off
+      bedroom_auto_track_switch:
```
Comment (JurajNyiri) [inline examples/EXAMPLES_ENTITIES.md:41]: Please remove _switch from the name, its already under switch domain

File: examples/EXAMPLES_ENTITIES.md [hunk 3]
```diff
@@ -27,6 +31,47 @@ switch:
...
+              entity_id: camera.bedroom_hd
+              auto_track_mode: 'on'
+        turn_off:
+          - service: tapo_control.set_auto_track_mode
+            data:
+              entity_id: camera.bedroom_hd
+              auto_track_mode: 'off'
+        icon_template: "mdi:radar"
+      # This is a switch to turn the alarm on/off
+      bedroom_alarm_switch:
```
Comment (JurajNyiri) [inline examples/EXAMPLES_ENTITIES.md:56]: Please remove _switch from the name, its already under switch domain

**Review by JurajNyiri — CHANGES_REQUESTED** [review]
Looks good, just some minor comment to change 👍

File: examples/EXAMPLES_ENTITIES.md (hunk 3 above)
Comment (david-kalbermatten) [reply examples/EXAMPLES_ENTITIES.md]: resolved with 3f9f831

**Review by JurajNyiri — APPROVED** [review]

**Pull request merged**